Home.html

```diff
@@ -136,13 +136,13 @@
 <body>
   <!-- navbar starts -->
   <header>
-    <div class="navbar"> 
+    <div class="navbar">
       <div class="main-heading">
 
         <div class="logo">
           <a href="#" class="endlogo">
             <img src="./Images/flag.png">
-           
```

Comment:
@Albert-Jokelin there are so many newline added instead of previous trailing spaces please revert this